hub/executors/encoders/image/facenet/manifest.yml

```diff
@@ -0,0 +1,8 @@
+name: torch facenet
+description: an image with the encoder of facenet pytorch implementation
+author: Shivam Raj (connect2shivamraj@gmail.com)
+url: NA
+documentation: https://github.com/jina-ai/jina-hub
+version: 0.0.3
+vendor: Jina AI Limited
+license: apache-2.0
```

Comment:
missing `update` key. Would you please have a look at [`update`](https://github.com/jina-ai/jina-hub#schema-of-manifestyml) in the documentation?

Reply:
@nan-wang Heyy,
I added the update field.
Should I publish the image to DockerHub myself ?

Reply:
No worry about that. Our CI/CD process will take care of it. ;)